notebooks/240-efficientdet-object-detection/240-efficientdet-object-detection.ipynb

```diff
@@ -0,0 +1,666 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        if (img.shape[-1]) == 1:
</pre><p>We don't assume there are grayscale images. Let's use KISS rule.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/871/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
I added this because there are some grayscale images in the COCO validation dataset, which were causing errors while validating the model. so should I skip grayscale images while validating?

Reply:
I see. I think we can assume we will use BGR images only